### DISCUSSION TITLE
F1 vm (virtual android) termux

### DISCUSSION THREAD

**Discussion by vishnu948** [termux/termux-app#2229] [Q&A] [answered]
Can we use termux in f1 vm

**Comment by agnostic-apollo** [comment, marked as answer]
No, f1 is not supported by termux and has issues with `apt`.

**Comment by ghost** [comment]
> > Can we use termux in f1 vm
> 
> Plz help me guys .. 

**No, it's not recommended to use termux as f1vm**

```
Because, it has a apt issues in termux in f1vm!
```
